src/monmove.cpp

```diff
@@ -1474,12 +1474,19 @@ bool monster::move_to( const tripoint &p, bool force, const float stagger_adjust
     }
 
     if( has_flag( MF_DRIPS_NAPALM ) ) {
-        if( one_in( 10 ) ) {
-            g->m.add_item_or_charges( pos(), item( "napalm" ) );
+        if( true ) { // one_in( 10 ) // removed randomness to test
```

Comment:
```cpp
if( true )
```
I know, it's a draft, but do not do it, please.

Reply:
:) Normally I wouldn't have pushed with that. But I was seeking help with the compiling and needed the code pushed to continue from home computer. Point taken.